modules/fundamental/src/sbom/model/mod.rs

```diff
@@ -23,6 +25,24 @@ pub struct SbomHead {
     pub name: String,
 }
 
+impl SbomHead {
+    pub async fn from_entity(
+        sbom: &sbom::Model,
+        sbom_node: Option<sbom_node::Model>,
+        _tx: &ConnectionOrTransaction<'_>,
+    ) -> Result<Self, Error> {
+        Ok(Self {
+            id: sbom.sbom_id,
```

Comment:
for my own education, why is this not cloned and the others are ... thx for answering my rust newbie questions

Reply:
> for my own education, why is this not cloned and the others are ... thx for answering my rust newbie questions

The type of `sbom_id`, `Uuid`, implements the `Copy` trait, so it will be copied when passed. The others don't implement `Copy`, so they need to be cloned in order to avoid a compiler error about "moving" the field.

Reply:
aha ! thx for connecting the dots!